presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -237,24 +237,41 @@ public String getSlug()
         return slug;
     }
 
+    /**
```

Comment:
Lets skip all the @ return comments for getter functions, it seems repetitive and is also left empty for many instances

Reply:
Actually - why not fill in the missing returns? It will lead to cleaner javadoc when generated?

Reply:
> It will lead to cleaner javadoc when generated?
IMO the utility of this documentation is more geared towards developers perusing the code through an IDE - in which case havingfor *some* of these simple getter methods, it is a bit of an overkill.

Reply:
My 0.02: let's skip the returns.  They basically re-state the method name in almost all cases.  Even for the generated documentation, you can simply look at the method name to understand what it is returning.  Instead, let's only fill it in when there's a particular constraint that's not obvious.

Reply:
I've unassigned this until it's fixed, I don't think restating what's in the code in the comment is the example we want to set for new documentation.  Of course this is just my opinion and we could solicit feedback from others here as well.  Please ping me if it's addressed and ready for re-review, thanks!

Reply:
- I've removed the `@returns`
- Removed descriptions of obvious methods

Please take a look again and let me know any methods that will benefit from a more detailed javadoc